script/state.py

```diff
@@ -0,0 +1,45 @@
+import json
+import psycopg2
+hostname = 'postgres'
+database='geovisualizer'
+username='postgres'
+pwd=''
```

Comment:
What is the user of this pwd, when you are passing the blank value while creating connection